### ISSUE TITLE
FSRS scheduling cards to be due in the past before my last review

### ISSUE THREAD

**Issue by colingohmh** [open-spaced-repetition/fsrs4anki-helper#92]
Hi Jarrett,

Been using your FSRS helper for a while now and am a big fan!

Can I just check whether this is a bug regarding what I have encountered?

I reviewed this card yesterday (7th May) and had pressed the again button which should show me the card again in a few days. However, today when I ran the FSRS helper "reschedule cards reviewed in the last 7 days", it placed the card as due on 15th April (see attached screenshot). May I check if this is a bug? Because now my overdue interval on this rescheduled card would be higher, which would affect the subsequent interval if I answer it again. Do let me know if it isn't clear :)

<img width="879" alt="image" src="https://user-images.githubusercontent.com/74810571/236812737-46840e8e-1d1d-47ef-900c-a58cf3751385.png">

**Comment by L-M-Sherlock** [comment]
Could we know the card info before the rescheduling?

**Comment by colingohmh** [comment]
oh sorryyy, how do i retrieve the info before the rescheduling? is the log for the specific card kept somewhere?

**Comment by colingohmh** [comment]
This is another card before rescheduling and after rescheduling by the FSRS helper
Card was just reviewed on 11 May and I had pressed Good

Before rescheduling: Due 25 May
<img width="775" alt="image" src="https://github.com/open-spaced-repetition/fsrs4anki-helper/assets/74810571/a604bb69-d9cd-41dc-bb0a-a08d6d0da6fd">

After: Due 11 May
<img width="797" alt="image" src="https://github.com/open-spaced-repetition/fsrs4anki-helper/assets/74810571/6e5fb459-21bf-465c-8a8d-eafb64d082ee">

So after rescheduling, the FSRS helper had shown it as due again even though I had just reviewed it yesterday

**Comment by L-M-Sherlock** [comment]
Could you share this deck (including review data) with me? I need it to locate the bug.

**Comment by colingohmh** [comment]
Sure, shall I email it to you? Thanks so much for your help :)

Just as a side note, the scheduling problems seems to be rectified when I click "undo all rescheduling" and then re-schedule all cards in the deck, just that I have to undo and redo it everyday

**Comment by L-M-Sherlock** [comment]
> Sure, shall I email it to you? Thanks so much for your help :)

You can modify the xxx.apkg to xxx.zip and attach it by dragging & dropping here.

**Comment by L-M-Sherlock** [comment]
Do you use the latest helper add-on? And could you share the parameters that you used?

**Comment by L-M-Sherlock** [comment]
I reschedule the card in my Anki:

![image](https://github.com/open-spaced-repetition/fsrs4anki-helper/assets/32575846/5446ec3e-2c5a-43f1-bd2c-018622756cc8)

I haven't reproduced the bug.

**Comment by L-M-Sherlock** [comment]
I think this issue has been solved in #108.
